test/app/num_bindings.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+)
+
+var numPostMenuBindings int64 = -1
+var numChannelHeaderBindings int64 = -1
```

Comment:
Obviously not a fan of using globals like this, but I'm thinking the test app project should be re-written to use things like kv store to properly support this feature. In the meantime, this seemed fine since this is not a production application.